torchmetrics/functional/text/sacre_bleu.py

```diff
@@ -278,20 +278,22 @@ def _lower(line: str, lowercase: bool) -> str:
 
 
 def sacre_bleu_score(
-    translate_corpus: Sequence[str],
-    reference_corpus: Sequence[Sequence[str]],
+    preds: Union[None, Sequence[str]] = None,
+    target: Union[None, Sequence[Sequence[str]]] = None,
     n_gram: int = 4,
     smooth: bool = False,
     tokenize: Literal["none", "13a", "zh", "intl", "char"] = "13a",
     lowercase: bool = False,
+    translate_corpus: Union[None, Sequence[str]] = None,
+    reference_corpus: Union[None, Sequence[Sequence[str]]] = None,
```

Comment:
```suggestion
    translate_corpus: Union[None, Sequence[str]] = None,  # ToDo: remove in v0.8
    reference_corpus: Union[None, Sequence[Sequence[str]]] = None,  # ToDo: remove in v0.8
```
could be useful to not forget...